code/game/machinery/computer/arcade.dm

```diff
@@ -420,7 +420,6 @@
 			if(food <= 0)
 				dat += "<br>You ran out of food and starved."
 				if(emagged)
-					user.nutrition = 0 //yeah you pretty hongry
```

Comment:
A lot of these are removed just to add a marker for myself on the github to go back and rework them for the new system.